ps_linklist.php

```diff
@@ -103,6 +109,14 @@ public function uninstallTab()
         return $tab->delete();
     }
 
+    public function hookActionSymfonyModuleRoutes()
+    {
+        return [];
```

Comment:
Double return watch out

Reply:
this was a test for module route management, this implementation was know kept But thanks I forgot to remove it